ArduCopter/Copter.cpp

```diff
@@ -125,7 +125,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
     SCHED_TASK(auto_disarm_check,     10,     50,  27),
     SCHED_TASK(auto_trim,             10,     75,  30),
 #if RANGEFINDER_ENABLED == ENABLED
-    SCHED_TASK(read_rangefinder,      20,    100,  33),
+    SCHED_TASK(read_rangefinder,      100,    100,  33),
```

Comment:
I don't think we will get away with a straight change to 100 Hz.  I think the best we can hope for is to make this a param value that defaults to 20.